target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4019-a62.dts

```diff
@@ -215,7 +215,7 @@
 };
 
 &pcie_bridge0 {
-	wifi@0,0 {
+	wifi@1,0 {
 		compatible = "qcom,ath10k";
 		reg = <0x00010000 0 0 0 0>;
```

Comment:
These node names should be correct, see comment https://github.com/openwrt/openwrt/pull/18843#discussion_r2117566415.